cmd/watt/aggregator.go

```diff
@@ -30,7 +30,7 @@ type aggregator struct {
 	snapshots chan<- string
 	// We won't consider ourselves "bootstrapped" until we hear
 	// about all these kinds.
-	requiredKinds       []string
+	requiredKinds       map[string]bool
```

Comment:
What's the new semantics associated with this field? Could you update the comment to reflect that?

Reply:
Done.